src/ui/public/embeddable/embeddable_handlers_registry.js

```diff
@@ -0,0 +1,9 @@
+import { uiRegistry } from 'ui/registry/_registry';
```

Comment:
Can we put this in `ui/registry` with the rest of the registries? I'm not a huge fan of the decision to go that route but we should probably stay consistent.